package/scripts/presto_client.py

```diff
@@ -80,7 +80,8 @@ def ensure_nodes_are_up(client):
         raise RuntimeError('Presto server failed to start within \
 {0} seconds.'.format(RETRY_TIMEOUT))
 
-
+# This class was copied more or less verbatim from
```

Comment:
where are you using this module? sorry I dont exactly remember when you added  this